src/commons/Config.ts

```diff
@@ -18,10 +19,18 @@ class Config {
     this.setConfig({});
   }
 
-  public setConfig(options: ConfigOptions) {
+  public async setConfig(options: ConfigOptions) {
     const {usePlatformColors = false, appScheme = 'light'} = options;
     this.usePlatformColors = usePlatformColors;
-    this.appScheme = appScheme;
+    this.appScheme = await AsyncStoragePackage?.getItem?.('appScheme') || appScheme;
```

Comment:
You don't check here if AsyncStoragePackage (you check only in setLocalScheme) so technically this would throw error for all users who don't have this package. 
I would create a small function here for validating the package so you won't have to do it many times